clusters/hosted-mgmt/hive/pools/multiarch/admins_multiarch-cluster-pool_rbac.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: multiarch-cluster-pool
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: multiarch-pool-admins
+    namespace: multiarch-cluster-pool
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: hive-cluster-pool-admin
+  subjects:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: Group
+    name: multiarch-pool-admins
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: multiarch-pool-admins
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: cluster-namespace-view
+  subjects:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: Group
+    name: multiarch-pool-admins
```

Comment:
do we have this group in rover?

Reply:
thanks, I miss this part, fixing

Reply:
created group multiarch-pool-admins in rover